Makefile

```diff
@@ -266,7 +266,7 @@ ifeq (,$(shell which opm 2>/dev/null))
 	set -e ;\
 	mkdir -p $(dir $(OPM)) ;\
 	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.31.0/$${OS}-$${ARCH}-opm ;\
+	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.29.0/$${OS}-$${ARCH}-opm ;\
```

Comment:
Why the curl downgrade? The reasoning should be in the commit.

Reply:
It is I believe.

```
Additionally fixes the opm image tag to one that exists.
```

Reply:
@jpodivin WDYT? Can we move forward with merging this one?

Reply:
I suppose that makes sense. It may have been better as a separate patch but if that's not possible, why not.